mobilizon_reshare/dataclasses/__init__.py

```diff
@@ -0,0 +1,9 @@
+from mobilizon_reshare.dataclasses.event import _MobilizonEvent
+from mobilizon_reshare.dataclasses.event_publication_status import (
+    _EventPublicationStatus,
+)
+from mobilizon_reshare.dataclasses.publication import _EventPublication
+
+EventPublication = _EventPublication
+MobilizonEvent = _MobilizonEvent
+EventPublicationStatus = _EventPublicationStatus
```

Comment:
these are not bindings but a pattern to expose only the dataclasses we care about and that should be used from the outside. 

It's still not final and some parts of the code are breaking the pattern but I needed it to avoid some circular imports.

Reply:
i'm pretty sure assignment binds a value to a name, i was asking because the syntax is quite similar (but I can see the semantics isn't) to hibernate metamodels ( https://hibernate.org/orm/tooling/ ) which are in fact generated.

anyway LGTM when you fix the pipeline we can merge this